### PR TITLE
Review of SOP for Clinical Validation of Mappings

### PR DESCRIPTION
**Pull request by p-talapova**
This pull request introduces SOP for the Clinical Validation of Mappings.

Additionally, I have linked a PNG image that visually represents some of the key workflows described in the text. While the link serves its purpose, integrating this image directly into our general UI might provide a more seamless experience for users. I am open to suggestions on how we can better incorporate visual aids into the documentation.

Looking forward to your feedback!

### CODE REVIEW

File: sop-website/docs/OMOP-Mapping/Mappings-Clinical-Validation.mdx
Comment (hol7001) [inline sop-website/docs/OMOP-Mapping/Mappings-Clinical-Validation.mdx:1]: The SOP for Clinical Validation of Mappings looks great. A couple of feedbacks:
1. Do we have an overall process diagram so we can demonstrate the relationship of this clinical validation of mappings in the overall process diagram? 
2. When involving expert judgement, we usually need to assess baseline agreement among experts. Is a single expert enough for the task? Or we need double blind validation?

File: sop-website/docs/OMOP-Mapping/Mappings-Clinical-Validation.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## STEP 2: SET UP ENVIRONMENT
+Grant permissions for the GitHub repository and the Google Drive directory to the relevant team members. Additionally, assist with the setup of the working environment as needed. Ensure that the expert is informed about the clinical validation [Guide](https://github.com/chorus-ai/chorus-mapping/tree/mapping_dev/Docs/MappingClinicalValidation) available here, on GitHub. It is crucial that they familiarize themselves with this guide to streamline the validation process.
+
+## STEP 3: COMPARE SOURCE AND TARGET
+The Clinical Expert compares the semantic content of two fields: "source_description" and "target_concept_name". Their evaluation will determine the extent to which these fields convey the same clinical concept, despite potential differences in terminology, syntax, or abbreviation.
+### Scenarios for Decision Making
+1) **Equivalent Concepts and Correct Mapping**: If the expert concludes that both fields articulate the same clinical idea, despite variations in wording such as synonyms, acronyms, or word order, they should enter "1" in the "decision" field.
```
Comment (hol7001) [inline sop-website/docs/OMOP-Mapping/Mappings-Clinical-Validation.mdx:25]: the same clinical idea to the same clinical meaning.

File: sop-website/docs/OMOP-Mapping/Mappings-Clinical-Validation.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+2) **Divergent Concepts and Incorrect Mapping**: If the expert finds that the fields convey different meanings, suggesting an erroneous mapping, they should enter "2" in the "decision" field.
+3) **Introduction of New Mapping**: If the expert proposes a new mapping, they should add it by inserting a new row(s) accordingly.
+
+## STEP 4: VALIDATE MAPPING
+The Clinical Expert documents their decision in designated fields: 'review_date' captures the date of the review, 'reviewer_name' records the expert’s name, 'reviewer_comment' includes any additional notes, and 'orcid_id' provides the expert's unique identifier.
```
Comment (hol7001) [inline sop-website/docs/OMOP-Mapping/Mappings-Clinical-Validation.mdx:30]: Some grammar problem in Line 30. Also Maybe show an example here?

File: sop-website/docs/OMOP-Mapping/Mappings-Clinical-Validation.mdx
Comment (p-talapova) [reply sop-website/docs/OMOP-Mapping/Mappings-Clinical-Validation.mdx:1]: Hi there,

1) Indeed, we have an overarching process diagram. Thank you for the suggestion - it is a great idea to illustrate where Clinical Validation fits within this process. I will add such a visualization.

2) That's an excellent question. Double-blind validation is undoubtedly beneficial, but due to constraints of time and cost, I recommend implementing it after the initial validation phase. This secondary validation will address terms that did not receive a clear verdict initially. If this approach is agreeable, I will include a statement in the SOP to clarify the steps following the initial validation.

Comment (hol7001) [reply sop-website/docs/OMOP-Mapping/Mappings-Clinical-Validation.mdx:1]: Thank you.

**Review by hol7001 — APPROVED** [review]
The diagram is great.  Thanks.

**Pull request merged**